app/search_builders/curation_concerns/search_builder.rb

```diff
@@ -31,22 +31,4 @@ def discovery_permissions
     return ['edit'] if blacklight_params[:works] == 'mine'
     super
   end
-
-  # This is included as part of blacklight search solr params logic
-  def filter_models(solr_parameters)
-    solr_parameters[:fq] ||= []
-    solr_parameters[:fq] << '(' + (work_clauses + collection_clauses).join(' OR ') + ')'
-  end
-
-  def work_clauses
-    return [] if blacklight_params.key?(:f) && Array(blacklight_params[:f][:generic_type_sim]).include?('Collection')
```

Comment:
I think we may have wanted to keep this. to power https://github.com/projecthydra-labs/curation_concerns/blob/master/app/helpers/curation_concerns/search_paths_helper.rb#L2-L10  It's interesting that no specs are failing though.